### ISSUE TITLE
IDEX board and PT100

### ISSUE THREAD

**Issue by sllo13** [bigtreetech/BTT-E3-RRF#20]
How to I configure reprap to use a PT100 thermistor with the IDEX board
how to I set the thermistor in the config.g file?
Searched everywhere and havent found an answer

**Comment by Ragpuss** [comment]
I would like to know the same thing.
Did you manage to find any answers ?
Also should the heater plug into the IDEX board if using a PT100 on the same board ?
It's strange that there's still no info for this !!

**Comment by chef79** [comment]
would love some info about this to

**Comment by Ragpuss** [comment]
It looks to me like BTT have combined their PT100 daughter board into the IDEX so if thats the case it should be a setting of 20 for the sensor, I am still not sure on the pin.h file if it will automatically push the connection to the IDEX board like it does for the 2nd Z stepper or if the correct pin assignment has to be put somewhere.

**Comment by chef79** [comment]
yes there is pt100 chip on the idex,  but what table do one use to get correct readings.
lowered resistance to 1400 and got 23C  heated the sensor to 60c and temp dropped to 16c hehe.

**Comment by chef79** [comment]
heat.tempSensePins = {bedtemp,PT100}; 
you got this in board.txt?  i needed to change E0 to PT100 to get it working

**Comment by Ragpuss** [comment]
Isn't E0 the hot end heater ? , it would need to be the thermistor to change.
I haven't changed any pin assignments yet as it's not something that I have messed with much in the past and don't wont to blow my printer up, lol.
Did you have it set to 20 when you got those odd readings. If the IDEX is using 3.3v instead of 5v it might need to be 21
I noticed a jumper that's marked as 3.3v with a jumper cap on it, not sure if that's for the PT100 or not.

**Comment by marcoeli007** [comment]
Define in the file "pins_BTT_E3_RRF.h" TEMP_0_PIN to FPC8_PIN. Disable PA0
![image](https://user-images.githubusercontent.com/57828489/210572955-32cb39a9-91e7-4874-b5ad-b8536068bf7d.png)

Set in "Configuration.h" TEMP_SENSOR_0 to 20
![image](https://user-images.githubusercontent.com/57828489/210572764-79de30a2-7d1d-4b73-9c82-b3efdaf63835.png)
